apps/src/blockTooltips/DropletFunctionTooltip.html.ejs

```diff
@@ -15,7 +15,13 @@
     <%= getPrefixedName() %>(<% for (var i = 0; i < locals.parameters.length; i++) { -%><%- locals.parameters[i].name -%><% if (i < locals.parameters.length - 1) { -%>, <% } -%><% } -%>)
   <% } %>
 </div>
-<% if (locals.functionShortDescription) { %><div><%= locals.functionShortDescription %></div><% } %>
+<% if (locals.functionShortDescription) { %>
+  <% locals.functionShortDescription.split('\n').map(function(descriptionLine) { %>
+    <div>
+      <%= descriptionLine %>
+    </div>
+  <% }) %>
+<% } %>
```

Comment:
.ejs 😭

If it's not already tracked, can you add an item to the backlog to get these views converted over to React?